felix/config/config_params.go

```diff
@@ -330,7 +330,8 @@ type Config struct {
 	// - calicoIPAM: use IPAM data to contruct routes.
 	RouteSource string `config:"oneof(WorkloadIPs,CalicoIPAM);CalicoIPAM"`
 
-	RouteTableRange idalloc.IndexRange `config:"route-table-range;1-250;die-on-fail"`
+	RouteTableRange  idalloc.IndexRange `config:"route-table-range;1-250;die-on-fail"`
+	RouteSyncEnabled bool               `config:"bool;true"`
```

Comment:
I would suggest to use `RouteSyncDisabled` with default value `false`. The configuration item is part of FelixConfiguration resource and should be omit-empty.